Makefile

```diff
@@ -62,6 +62,9 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT="true" KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v ./... -coverprofile cover.out -timeout 1800s
 
+.PHONY: lint
+lint: fmt vet generate helm ## Test formatting, uncommitted and generated files.
```

Comment:
This stuff is exactly linter - it detects stupid errors and inconsistens is code.

Reply:
My point is it common to have `make lint`, `make fmt` in Makefile and expect that one would run code linters and other one would format the code. And it would be really odd to run `make lint` and have code reformatted and generators being run, because you expect a linter to be read-only operation.

Reply:
Let's revisit this later when somebody want to integrate better linter. For now this is best what we have.

Reply:
But you expect `make build` to generate/change things, not `make lint`. Though I'm not sure it shouldn't fmt/vet also.